lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -25,7 +25,9 @@ def receive(event)
 
     # Receive an array of events and immediately attempt to index them (no buffering)
     def multi_receive(events)
-      retrying_submit(events.map {|e| event_action_tuple(e) })
+      events.each_slice(@flush_size) do |slice|
```

Comment:
Shouldn't there be a test if `batch_size > @flush_size` and then slice it? Or does `each_slice` takes care of that internally using the `size` of `events` ?